core/site_scons/tools.py

```diff
@@ -1,11 +1,46 @@
-
 def add_font(font_name, font, defines, sources):
     if font is not None:
         defines += [
             'TREZOR_FONT_' + font_name + '_ENABLE=' + font,
             'TREZOR_FONT_' + font_name + '_INCLUDE=\\"' + font.lower() + '.h\\"',
-            ]
+        ]
         sourcefile = 'embed/extmod/modtrezorui/fonts/' + font.lower() + '.c'
         if sourcefile not in sources:
             sources.append(sourcefile)
 
+
+def get_hw_model_as_number(hw_model):
+    return int.from_bytes(hw_model.encode(), 'little')
```

Comment:
Just to make sure - the resulting binary fill will contain `T2B1` not `1B2T`, right?

Reply:
yes.

ie for 'T2T1' this will convert to number 827601492, or 0x31543254, which is then stored on little endian arm in memory in this order: 0x54, 0x32, 0x54, 0x31, which is also the order how "T2T1" string would be stored.